okhttp/src/main/java/okhttp3/Cookie.kt

```diff
@@ -39,6 +39,7 @@ import java.util.regex.Pattern
  *
  * [chromium_extension]: https://code.google.com/p/chromium/issues/detail?id=232693
  */
+@Suppress("DataClassPrivateConstructor")
```

Comment:
Don't suppress this one? It's sweeping a real problem under the rug and we haven't yet addressed that problem.